### PR TITLE
Add extendedTRS endpoints to openapi.yaml, minor changes to swagger

### PR DESCRIPTION
**Pull request by garyluu**
For dockstore/dockstore#3264

Dropped plain/text because...it's not plain/text

### CODE REVIEW

**Comment by denis-yuen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dockstore/dockstore/pullRequest?prid=5128816&bid=16855595)

**Comment by denis-yuen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dockstore/dockstore/pullRequest?prid=5128816&bid=16855595)

**Comment by denis-yuen** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/dockstore/dockstore/pullRequest?prid=5128816&bid=16855595)

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java [hunk 1]
```diff
@@ -62,10 +69,13 @@
     @GET
     @Path("/tools/{organization}")
     @UnitOfWork(readOnly = true)
-    @Produces({ "application/json", "text/plain" })
-    @ApiOperation(value = "List tools of an organization", notes = "This endpoint returns tools of an organization. ", response = ToolV1.class, responseContainer = "List")
+    @Produces({ MediaType.APPLICATION_JSON })
```
Comment (coverbeck) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java]: You dropped `text/plain` -- is that on purpose?

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java [hunk 2]
```diff
@@ -111,10 +130,13 @@ public Response workflowsOrgGet(
     @GET
     @Path("/containers/{organization}")
     @UnitOfWork(readOnly = true)
-    @Produces({ "application/json", "text/plain" })
-    @ApiOperation(value = "List entries of an organization", nickname = "entriesOrgGet", notes = "This endpoint returns entries of an organization. ", response = ToolV1.class, responseContainer = "List")
+    @Produces({ MediaType.APPLICATION_JSON })
```
Comment (coverbeck) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java]: Dropped text/plain

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java [hunk 3]
```diff
@@ -124,10 +146,13 @@ public Response entriesOrgGet(
     @GET
     @Path("/organizations")
     @UnitOfWork(readOnly = true)
-    @Produces({ "application/json", "text/plain" })
-    @ApiOperation(value = "List all organizations", nickname = "entriesOrgsGet", notes = "This endpoint returns list of all organizations. ", response = String.class, responseContainer = "List")
+    @Produces({ MediaType.APPLICATION_JSON })
```
Comment (coverbeck) [inline dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java]: Ditto

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java (hunk 2 above)
Comment (garyluu) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java]: Yup, did that on purpose

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java (hunk 1 above)
Comment (garyluu) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java]: yup

File: dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java (hunk 3 above)
Comment (garyluu) [reply dockstore-webservice/src/main/java/io/dockstore/webservice/resources/proposedGA4GH/ToolsExtendedApi.java]: yup

**Pull request merged**